previousnext/agov
# 34723 - Re-create Whitlam Styleguide D8

Steps to test:

* Go to https://agov-pr649.agov.com.au/admin/appearance (admin:admin) and ensure that agov_whitlam is enabled by default and the title, description, and screenshot looks exactly how we want it to

* Test that front-end workflow generates styleguide, compiles CSS and injects it into Styleguide. Test that  superfish.min.js and jquery.min.js is being added to styleguide by visiting the network tab.
https://agov-pr649.agov.com.au/styleguide/agov_whitlam/section-components.html
